build/assets/scripts/runc-wrapper.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+# "0" means "disabled"
+if [ "{{.CPUMask}}" == "0" ]; then
+	exec /usr/bin/runc "$@"
+fi
+if [ -n "$3" ] && [ "$3" == "start" ]; then
+        if [ -n "$4" ]; then
+                bundle=$(/bin/runc state "$4" 2>/dev/null | jq .bundle -r)
```

Comment:
do we want to use `/usr/bin/runc` here? - in general we should always use the same path, probably `/bin/runc` is better

Reply:
Sure

Reply:
We need to call it from the "official" path. If /bin/runc is the place, let's do it

Reply:
And is the official runc path in the runtime_path , so 100%

Reply:
so we use `/bin/runc` everywhere?

Reply:
Yes, thank you